ipv8/test/attestation/trustchain/test_community.py

```diff
@@ -1,11 +1,11 @@
-from ....attestation.trustchain.block import TrustChainBlock
-from ....attestation.trustchain.caches import CrawlRequestCache
-from ....attestation.trustchain.community import TrustChainCommunity, UNKNOWN_SEQ
-from ....attestation.trustchain.listener import BlockListener
 from ...attestation.trustchain.test_block import TestBlock
 from ...base import TestBase
 from ...mocking.ipv8 import MockIPv8
 from ...util import twisted_wrapper
+from ....attestation.trustchain.block import TrustChainBlock
```

Comment:
Does this import order follow PEP8?

Reply:
Perhaps, as Pylint isn't complaining. However, I think it would indeed be best to group this with the other `....attestation.*` imports.